sections/testing/unit-testing.tex

```diff
@@ -0,0 +1,97 @@
+\section{Unit testing}
+Unit testing is our primary type of testing for the system, and belongs to the dynamic white-box testing category.
```

Comment:
```suggestion
Unit testing is our primary way of testing our system, and belongs in the dynamic white-box testing category.
```
Har givet et foreslag til hvordan sætningen kan laves om men er stadig ikke så glad for den, overvej og del den op i to sætninger